frontend/src/components/entity/EntityCard.tsx

```diff
@@ -38,20 +40,29 @@ const EntityCard = ({
   });
   const [settingsVisible, setSettingsVisible] = useState(!isSmallScreen);
 
+  const shouldDisplayScores = (
+    compact: boolean,
+    entity: Recommendation | RelatedEntityObject
+  ) => {
+    return !compact && 'tournesol_score' in entity ? true : false;
+  };
```

Comment:
With this function, Typescript is not able to check that `entity` is a `Recommendation` (with scores).  
To avoid using `as`, this check and the scores render could be defined in the same function:

```tsx
  const getScores = () => {
    if ('tournesol_score' in entity && !compact) {
      return <EntityCardScores entity={entity} />;
    }
    return null;
  };
  ```

Reply:
Thanks, fixed by https://github.com/tournesol-app/tournesol/pull/864/commits/a3b3ab46096252907c2f5ac1d69b997edf2aa1c9.